bundle/internal/schema/annotations.go

```diff
@@ -0,0 +1,130 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"reflect"
+	"strings"
+
+	"github.com/ghodss/yaml"
```

Comment:
You can use `yamlsaver` for saving to disk.

This gives you some control over the node style if you need it (e.g. use blocks for all descriptions).